semver/format_test.ts

```diff
@@ -77,7 +77,7 @@ Deno.test("format", async (t) => {
     await t.step({
       name: `format(${version} ${style} ${expected})`,
       fn: () => {
-        const v = parse(version)!;
+        const v = parse(version as SemVer)!;
```

Comment:
Why type cast to deprecated signature? I think we need to fix the test data instead